### PR TITLE
NMS-14279: Handle unknown flow direction

### PR DESCRIPTION
**Pull request by fooker**
### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-14279

### CODE REVIEW

**Comment by j-white** [comment]
Should we update flow thresholding here too? Had a patch for that here: https://github.com/OpenNMS/opennms/commit/1934bae9a0fd170bf5f09f6692f3b30cd9ffa9b2#diff-bacdc74287e5ac0fab0facb94ea6430af784e51621d2e145c1c7aca5283ed83dR361

**Comment by Naicisum** [comment]
@j-white / @fooker - Is there an associated PR for Nephron? 😁

**Comment by fooker** [comment]
> @j-white / @fooker - Is there an associated PR for Nephron? grin

Not for now. There is https://issues.opennms.org/browse/NMS-14477 to track that.

**Review by j-white — APPROVED** [review]

**Review by christianpape — CHANGES_REQUESTED** [review]
Minimal changes...

File: features/flows/elastic/src/main/java/org/opennms/netmgt/flows/elastic/thresholding/FlowThresholding.java [hunk 1]
```diff
@@ -66,6 +66,7 @@
 import org.opennms.netmgt.threshd.api.ThresholdInitializationException;
 import org.opennms.netmgt.threshd.api.ThresholdingService;
 import org.opennms.netmgt.threshd.api.ThresholdingSession;
+import org.postgresql.shaded.com.ongres.scram.common.bouncycastle.pbkdf2.Integers;
```
Comment (j-white) [inline features/flows/elastic/src/main/java/org/opennms/netmgt/flows/elastic/thresholding/FlowThresholding.java:69]: Bad import?

Comment (fooker) [reply features/flows/elastic/src/main/java/org/opennms/netmgt/flows/elastic/thresholding/FlowThresholding.java]: Ooops

**Review by j-white — APPROVED** [review]

**Pull request merged**